core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -427,13 +427,8 @@ void compact(String tableName, Text start, Text end, List<IteratorSetting> itera
       throws AccumuloSecurityException, TableNotFoundException, AccumuloException;
 
   /**
-   * Starts a full major compaction of the tablets in the range (start, end]. If the config does not
-   * specify a compaction strategy, then all files in a tablet are compacted. The compaction is
-   * performed even for tablets that have only one file.
-   *
-   * <p>
-   * Only one compact call at a time can pass iterators and/or a compaction strategy. If two threads
-   * call compaction with iterators and/or a compaction strategy, then one will fail.
+   * Starts a full major compaction of the tablets in the range (start, end]. Files to be compacted
+   * will be selected by the compaction service configured on the table.
```

Comment:
@dlmarion I looked at the code in the manager to see what would happen and wrote the following javadoc as a result of that.  I was not able to validate the javadoc as my dev env is a bit out of sorts at the moment.

```suggestion
  * Starts a full major compaction of the tablets in the range (start, end]. If the config does not
   * specify a compaction selector (or a deprecated strategy), then all files in a tablet are compacted. The compaction is
   * performed even for tablets that have only one file.
   *
   * <p>
   * The following optional settings can only be set by one compact call per table at the same
   * time.
   *
   * <UL>
   *     <LI>Execution hints : {@link CompactionConfig#setExecutionHints(Map)}</LI>
   *     <LI>Selector : {@link CompactionConfig#setSelector(PluginConfig)}</LI>
   *     <LI>Confgigurer : {@link CompactionConfig#setConfigurer(PluginConfig)}</LI>
   *     <LI>Iterators : {@link CompactionConfig#setIterators(List)}</LI>
   *     <LI>Compaction strategy :
   *     {@link CompactionConfig#setCompactionStrategy(CompactionStrategyConfig)}</LI>
   * </UL>
   *
   * <P>If two threads call this method concurrently for the same table and set one or more of the
   * above then one thread will fail.
```